addons/website_slides_survey/tests/test_course_certification_unlink.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo.tests import tagged
+from odoo.exceptions import ValidationError
+from odoo.addons.survey.tests.common import TestSurveyCommon
+
+
+@tagged('unlink')
+class TestSurvey(TestSurveyCommon):
```

Comment:
You could also inherit from `SlidesCase` sorry

Reply:
It creates an error when you inherit from two classes. We have a ValidationError that states that we have two different users with the same ID which is impossible. 
=> Impossible to inherit from both classes